### PR TITLE
TSV export for record / node 

### PR DESCRIPTION
**Pull request by cgmeyer**
…nctions to export_record and export_node

### CODE REVIEW

File: gen3/submission.py [hunk 1]
```diff
@@ -69,49 +69,67 @@ def query(self, query_txt, variables=None, max_tries=1):
...
         Examples:
             This exports a single record from the sandbox commons.
 
-            >>> Gen3Submission.export_node("DCF", "CCLE", "d70b41b9-6f90-4714-8420-e043ab8b77b9")
+            >>> Gen3Submission.export_record("DCF", "CCLE", "d70b41b9-6f90-4714-8420-e043ab8b77b9","json")
 
         """
-        api_url = "{}/api/v0/submission/{}/{}/export?ids={}&format=json".format(
-            self._endpoint, program, project, uuid
+        api_url = "{}/api/v0/submission/{}/{}/export?ids={}&format={}".format(
```
Comment (zflamig) [inline gen3/submission.py]: Can we add an assert that fileformat == 'json' or 'tsv' before this since those are the only two supported types?

**Comment by zflamig** [comment]
I agree with @mfshao that the function shouldn't write to a file... we can have an example of how to write out a TSV to disk separately from what the function actually does?

**Comment by zflamig** [comment]
Forgot to add, once we merge this we should make a new version and bump the major version. This is a breaking change since we change the function names

File: gen3/submission.py [hunk 2]
```diff
@@ -262,3 +276,11 @@ def get_graphql_schema(self):
         output = requests.get(api_url).text
         data = json.loads(output)
         return data
+
+def export_file(filename, output):
```
Comment (mfshao) [inline gen3/submission.py]: should this be a private method `_export_file(self, filename, output)` rather than a plain function?

Comment (zflamig) [reply gen3/submission.py]: yes please

**Pull request merged**